src/main/java/uk/gov/hmcts/divorce/common/model/Application.java

```diff
@@ -41,7 +42,7 @@
 
     @CCD(
         label = "Has applicant 2's marriage broken down irretrievably?",
-        access = {DefaultAccess.class}
+        access = {DefaultAccess.class, Applicant2Access.class}
```

Comment:
I think you should be able to remove DefaultAccess here as the [CREATOR] will never write this field